src/services/beerOrder.service.js

```diff
@@ -0,0 +1,27 @@
+//Delete later
+const BeerOrder = require("../models/beerOrder.model");
+
+class BeerOrderService {
+  async create(order) {
+    const newBeerOrder = new BeerOrder(order);
+    console.log(newBeerOrder);
+    await newBeerOrder.save();
+    return newBeerOrder;
+  }
+
+  async findById(orderId) {
+    const beerOrder = await BeerOrder.findOne({ _id: orderId });
+    return beerOrder;
+  }
+
+  async findAll() {
+    const allBeerOrders = await BeerOrder.find();
+    return allBeerOrders;
```

Comment:
si la beer order tiene un id de la persona sería bueno hacer un populate como
`return BeerOrder.find()
        .populate('person')`, check [this](https://mongoosejs.com/docs/populate.html)

Reply:
Na mas pon esto

Reply:
Lo woa checar pero es lo minimo posible para que levante, recordais?